client/app/bundles/course/assessment/pages/AssessmentStatistics/StudentAttemptCountTable.tsx

```diff
@@ -22,20 +26,54 @@ import { getAssessmentStatistics } from './selectors';
 import translations from './translations';
 import { getJointGroupsName, translateStatus } from './utils';
 
+interface StudentSubmissionChipProps {
+  className?: string;
+  linkTo?: string;
+  workflowState?: WorkflowState | typeof workflowStates.Unstarted;
+}
+
+const StudentSubmissionChip: FC<StudentSubmissionChipProps> = (props) => {
```

Comment:
Duplicated code in `StudentMarksPerQuestionTable.tsx`, so should extract this as a common component and use sitewide.  
Ideally check if we can also replace chip in `/submissions` page `client/app/bundles/course/assessment/submission/pages/SubmissionsIndex/SubmissionsTableRow.jsx` L190